src/ray_agents/cli/templates/agent/README.md

```diff
@@ -98,6 +102,51 @@ class ChatbotAgent(RayAgent):
         return {"response": "Hello!"}
 ```
 
+## Resource Configuration
+
+Configure CPU, GPU, memory, and replicas for your agents using two methods:
+
+### Method 1: Ray Decorator (Set Defaults in Code)
+
+Use the `@ray.remote` decorator to set default resource requirements for your agent:
+
+```python
+import ray
+from ray_agents import RayAgent
+
+@ray.remote(num_cpus=2, num_gpus=0, memory=4 * 1024**3)
+class MyAgent(RayAgent):
+    def __init__(self):
+        super().__init__()
+
+    def run(self, data: dict) -> dict:
+        return {"response": "Hello!"}
+```
+
+**Note:** The `memory` argument requires bytes (e.g., `4 * 1024**3` for 4GB).
+
+### Method 2: CLI Flags (Runtime Overrides)
+
+Override resource settings at runtime using CLI flags:
+
+```bash
+rayai serve --my-agent-num-cpus=4 --my-agent-memory=8GB --my-agent-num-replicas=2
```

Comment:
This seems non standard. The more I think about it I am leaning towards using KubeRay to configure services. That will allow us to enable developers to deploy on any cloud, and is the most scalable/standard way to deploy Ray in production at any scale.